HT2

eumakarov opened the pull request:
Added second HT + GitHub Pages link

eumakarov committed:
HW2

eumakarov committed:
Update README.md

anastasiatol reviewed (changes requested):
Задать контенту maxWidth 
post_text не всегда весь текст влазит, высота должна блока должна формироваться размером контента или многоточие вместо того, что не влезло
viewport не задан

Много вопросов по бэму. У тебя блок_элемент, а модификаторов не было?
Блок не получает в стилях внешнюю геометрию и позиционирование (margin или position)
Нарушается в классах: 
search-form
navigation
article
Post и др.
Присваивай второй класс как например search-form она и блок сама по себе и может выступать элементом header. Поэтому просто добавляешь класс header_search-form и вот на это уже можно вешать эти свойства.

Селекторы типа 
#id
.navigation_menu > li 
a:not(.rectangle_media)
.submenu li
.navigation_menu > li:hover .submenu
H4
в БЭМе не используются, задаются классы

.post:first-child  не всегда же будет, что именно первый пост отличаться цветом. Здесь нужен модификатор

Добавить кармашек под heder полукруглый, 
между меню вертикальные полоски высотой с текст, 
в секции about me текст при некоторой ширине экрана значительно выступает ниже картинки, 
при узком окне navigation_menu (когда становится вертикальным) занимает много места

eumakarov committed:
fixed visual components

eumakarov committed:
last fixes

БЭМ теперь БЭМ
Визуальные исправления
Убраны неиспользуемые участки

anastasiatol reviewed (changes requested):
По БЭМу почти хорошо, но осталось 2 момента:
Блоку грид эриа не задавать. Display: grid можно, но когда мы определяем grid-area мы определяем его местоположение, что не хорошо для блока
Модификаторы, первый пост имеет класс .article__post_first
Когда используем модификатор, всегда будет присвоено 2 класса без модификатора и с ним. (у тебя там в сумме 3 получится), добавляешь еще .article__post. Из .article__post_first удалить повторяющиеся свойства. 
Вот тут пример можно посмотреть:
https://ru.bem.info/methodology/css/#%D0%9C%D0%BE%D0%B4%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%82%D0%BE%D1%80%D1%8B 

Баги: 
На мобильных устройствах появляется горизонтальный скрол.
Хотелось бы заголовок над контентом, так слишком сбоку. (1скрин)
Баг с верхней навигацией. (2 и 3 скрин)
![1](https://user-images.githubusercontent.com/30156652/51940199-685e2880-2422-11e9-89df-9a7b89d5e68d.jpg)
![2](https://user-images.githubusercontent.com/30156652/51940200-685e2880-2422-11e9-872d-7edf271738ea.jpg)
![3](https://user-images.githubusercontent.com/30156652/51940202-685e2880-2422-11e9-8411-3e46c60edaed.jpg)

eumakarov committed:
fixes

Подправил все замечания кроме, пожалуй, самого непонятного для меня - горизонтального скролла на мобильных устройствах

eumakarov committed:
Delete styles.css

eumakarov committed:
Delete index.html

eumakarov committed:
reupload

Забросил изменения не в ту папку ¯ \ _ (ツ) _ / ¯

eumakarov committed:
last fixes

Исправил горизонтальный скролл на мобильных устройствах

anastasiatol reviewed (approved):
При узком экране/на мобильных устройствах:
в статьях расстояние от заголовка до картинки в первой статье больше чем в остальных. 
кнопка more прилипает к краю экрана
